packages/ui/src/common/hooks/useNetwork.ts

```diff
@@ -0,0 +1,10 @@
+import { useLocalStorage } from './useLocalStorage'
+
+export type NetworkType = 'local' | 'olympia-testnet'
+
+export const useNetwork = () => {
+  const [network, setNetwork] = useLocalStorage<NetworkType>('network')
+  const resolvedNetwork = network || 'local'
+
+  return [resolvedNetwork, setNetwork] as const
```

Comment:
Why `as const` needed here?

Reply:
See this: [https://github.com/typescript-cheatsheets/react#custom-hooks](https://github.com/typescript-cheatsheets/react#custom-hooks)